chapters/results/results.tex

```diff
@@ -0,0 +1,7 @@
+\chapter{Results}
+\todo{Add some introduction.}
+The project that was created as the practical part of the thesis is the \textit{Hyper} video game available at \url{https://github.com/Non-Euclidean-World/Hyper}.
```

Comment:
I don't think I will stay availlable for long. I think we will have to make it private unless the University specifically allows us to make it public

Reply:
added a todo